nemo/collections/asr/data/audio_to_label.py

```diff
@@ -1318,3 +1318,87 @@ def __len__(self):
 
     def _collate_fn(self, batch):
         return _speech_collate_fn(batch, pad_id=0)
+
+
+class AudioPairToLabelDataset(AudioToSpeechLabelDataset):
+    @property
```

Comment:
This is a data class for supporting val EER calculation in training speaker verification model